src/TableRow.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { forwardRef } from 'react';
+import classNames from 'classnames';
+import { lighten } from 'polished';
+import createStyles from './createStyles';
+import { PropsFromStyles } from './types';
+
+const useStyles = createStyles(({ css, theme, color }) => ({
+  // table row base styles
+  root: css`
+    > :first-child {
+      min-width: 240px;
+    }
+  `,
+  hoverableRow: css`
```

Comment:
i would rename this to just `hoverable` since this style is in the context of a table row